### ISSUE TITLE
ATEM parameter resets on SuperConductor startup

### ISSUE THREAD

**Issue by Velikyi** [SuperFlyTV/SuperConductor#205]
**Describe the bug**
Not sure if this is a bug or intended, but nonetheless a result I would like to avoid:

If ATEM (have only tested with Constellation 2 M/E) is added as a bridge in SuperConductor, it messes up bunch of parameters, such as setting current program and preview to black, change all output routings and DVE settings to default etc. I believe it also changed multiview routing, but I could be wrong. It happens both initially when adding the bridge, as well as every time SuperConductor is started, regardless if any rundowns contain any parts using the ATEM bridge. This renders the entire ATEM bridge function completely unusable for us.

**To Reproduce**
1. Add an ATEM bridge
2. Open SuperConductor at any time after step 1

**Expected behavior**
I would expect it to do absolutely nothing, as with any other automation solution connected to ATEM when started. It should not in any way mess with the current state when only added as a connection, and only do so when programmed to do so, and only do exactly what I've programmed it to do.

**Desktop (please complete the following information):**
 - OS: Same with both MacOS 13.5 and Windows 10

**Additional context**
Same result with both of our ATEM Constellation 2 M/Es.

**Comment by nytamin** [comment]
Duplicate of https://github.com/SuperFlyTV/SuperConductor/issues/122
